apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -53,7 +53,14 @@
         , listener_authenticator_user/2
         ]).
 
--export([authenticator_examples/0]).
+-export([ authenticator_examples/0
```

Comment:
those examples function should export only test profile:
-ifdef(TEST).
-export([xxxx/0]).
-endif.